onnxruntime/core/common/path.cc

```diff
@@ -254,7 +254,11 @@ Path& Path::Append(const Path& other) {
 }
 
 Path& Path::Concat(const PathString& string) {
```

Comment:
what happens if `string` contains a directory separator? maybe components_ would have unexpected values.
also maybe `value` instead of `string`?

Reply:
one simple approach would be to enforce that no directory separators are present in the value to be concatenated

Reply:
Added a check to enforce that. Thanks.